tests/test_schema.py

```diff
@@ -124,6 +115,13 @@ def test_iter_errors_typeinvalid():
     assert error.value == '1'
 
 
+def test_minimal_takendown():
+    """Test that a valid minimal takedown passes validation."""
+    minimal_doc = {'action': 'takedown', 'amw_key': '123'}
+    actual_minimal_doc = schema.validate_schema(schema.takedown, minimal_doc)
+    assert actual_minimal_doc == minimal_doc
```

Comment:
This is a common pattern we use throughout our tests:

    assert actual == expected

You should rename 'minimal_doc` to `expected` and `actual_minimal_doc` to `actual`.